style.css

```diff
@@ -316,5 +316,145 @@ a {
     margin: 8px 20px;
   }
 
+  #contact {
+    padding: 80px 0 0 0;
+    background-color: #fff;
+  }
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 70px 0 0 0;
+    background-color: #6070ff;
+  }
+
+  footer h2 {
+    margin-top: 103px;
+    margin-bottom: 6px;
+    color: #fff;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #para {
+    color: #ebebff;
+    margin: 6px 24px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+  }
+
+  .name {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    margin: 87px 0 10px 0;
+    padding: 10px;
+    height: 80%;
+    width: 95%;
+  }
+
+  .email {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    padding: 10px;
+    margin: 10px 0;
+    height: 80%;
+    width: 95%;
+  }
+
+  .textarea {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    margin: 12px 0;
+    width: 100%;
+  }
```

Comment:
- [ ] It would be better to add padding to the `textarea` to improve your UI design.